src/DssSpell.t.sol

```diff
@@ -490,4 +592,431 @@ contract DssSpellTest is DssSpellTestBase {
+        rwaconduitout_014.hope(address(this));
+        assertEq(rwaconduitout_014.can(address(this)), 1);
+
+        rwaconduitout_014.kiss(address(this));
+        assertEq(rwaconduitout_014.bud(address(this)), 1);
+        rwaconduitout_014.pick(address(this));
+
+        uint256 pushAmount = 100 * WAD; // We push only 100 DAI on Görli
+        rwaconduitout_014.push(pushAmount);
+        rwaconduitout_014.quit();
+
+        assertEq(dai.balanceOf(address(rwaconduitout_014)), 0, "RWA014: Output conduit still holds Dai after quit()");
+        assertEq(psmGem.balanceOf(address(this)), pushAmount / daiPsmGemDiffDecimals, "RWA014: Psm GEM not sent to destination after push()");
+        assertEq(dai.balanceOf(address(rwaurn_014)), drawAmount - pushAmount, "RWA014: Dai not sent to destination after push()");
+
+        // as we have SF 0 we need to pay exectly the same amount of DAI we have drawn
+        uint256 daiToPay = drawAmount;
+
+        // Note: In the version of outputCounduit for this deal `push` is permissionles
+        // // wards
+        // GodMode.setWard(address(rwaconduitinurn_014), address(this), 1);
+        // // may
+        // rwaconduitinurn_014.mate(address(this));
+        // assertEq(rwaconduitinurn_014.may(address(this)), 1);
+
+        // transfer PSM GEM to input conduit
+        psmGem.transfer(address(rwaconduitinurn_014), pushAmount / daiPsmGemDiffDecimals);
+        assertEq(psmGem.balanceOf(address(rwaconduitinurn_014)), pushAmount / daiPsmGemDiffDecimals, "RWA014: Psm GEM not sent to input conduit");
+        
+        // input conduit 'push()' to the urn
+        rwaconduitinurn_014.push();
+
+        assertEq(dai.balanceOf(address(rwaurn_014)), daiToPay, "Balance of the URN doesnt match");
+
+        // repay debt and free our collateral
+        rwaurn_014.wipe(daiToPay);
+        rwaurn_014.free(1 * WAD);
+
+        // check if we get back RWA009 Tokens
+        assertEq(rwagem_014.balanceOf(address(this)), 1 * WAD, "RWA014: gem not sent back to the caller");
+
+        // check if we have 0 collateral and outstanding debt in the VAT
+        (ink, art) = vat.urns("RWA014-A", address(rwaurn_014));
+        assertEq(ink, 0, "RWA014: bad `ink` after free()");
+        assertEq(art, 0, "RWA014: bad `art` after wipe()");
+    }
+
+    function testFailRWA014_DRAW_ABOVE_LINE() public {
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        uint256 drawAmount = 500_000_001 * WAD;
+
+        // setting address(this) as operator
+        vm.store(address(rwaurn_014), keccak256(abi.encode(address(this), uint256(1))), bytes32(uint256(1)));
+
+        // Draw 2mm
+        rwaurn_014.draw(drawAmount);
+    }
+
+    function testFailRWA014_PUSH_ABOVE_LINE() public {
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        uint256 drawAmount = 1_000_000 * WAD;
```

Comment:
499 now and 2 later? Or do it however the test is supposed to be, but our line is not 1 or 2m, it is 500.

I think 500 and 1 later might also work